docs/visualization/.pages

```diff
@@ -5,19 +5,14 @@ nav:
         - Callback:
             - Freq/Period/Latency(Execution time): visualization/visualization_api/callback_information
             - Scheduling: visualization/visualization_api/callback_scheduling_visualization
-        - Node:
-            - Publish/Subscription Information: visualization/visualization_api/pub_sub_information
         - Communication:
             - Freq/Period/Latency(Execution time): visualization/visualization_api/communication_information
+            - Publish/Subscription Information: visualization/visualization_api/pub_sub_information
```

Comment:
https://github.com/tier4/CARET_doc/pull/98#pullrequestreview-1169670486

I think .pages files needs modification.
```
Previous: Visualize application structure
Next: Design
```
I think Next link should be `Basic APIs Concept`.
```
Previous: Visualize application structure
Next: Basic APIs Concept
```

Reply:
@hsgwa 
I modified `.pages` and construction to be correct `previous` and `next` link.